core/chains/evm/gas/models.go

```diff
@@ -380,11 +360,44 @@ func bumpDynamicFee(cfg Config, lggr logger.SugaredLogger, currentTipCap, curren
 	return DynamicFee{FeeCap: bumpedFeeCap, TipCap: bumpedTipCap}, nil
 }
 
-func getMaxGasPrice(userSpecifiedMax *assets.Wei, config Config) *assets.Wei {
-	return assets.WeiMin(config.EvmMaxGasPriceWei(), userSpecifiedMax)
+func bumpFeePrice(originalFeePrice *assets.Wei, feeBumpPercent uint16, feeBumpUnits *assets.Wei) *assets.Wei {
+	bumpedFeePrice := assets.MaxWei(
+		originalFeePrice.AddPercentage(feeBumpPercent),
+		originalFeePrice.Add(feeBumpUnits),
+	)
+	return bumpedFeePrice
 }
 
-func capGasPrice(calculatedGasPrice, userSpecifiedMax *assets.Wei, config Config) *assets.Wei {
-	maxGasPrice := getMaxGasPrice(userSpecifiedMax, config)
+func updateCurrentFeePrice(lggr logger.SugaredLogger, currentFeePrice *assets.Wei, bumpedFeePrice, maxGasPrice *assets.Wei) *assets.Wei {
```

Comment:
weird that githubs formatting broke here, are you perhaps missing a close curly brace?

Reply:
It seems fine on VSC, not sure why it breaks on github.